### PR TITLE
Added command line argument to restrict the number of rows processed

### PR DESCRIPTION
**Pull request by jp3477**

### CODE REVIEW

File: omop_file_validator.py [hunk 1]
```diff
@@ -898,4 +946,22 @@ def evaluate_submission(d):
...
+
+    parser.add_argument('-r',
+                        '--restrict',
+                        action='store_true',
+                        help="Flag to restrict rows by argument {--n}")
+    parser.add_argument('-n',
+                        '--n_rows',
+                        type=int,
+                        default=1000,
+                        help="Number of rows to validate per file.")
```
Comment (nishanthpp93) [inline omop_file_validator.py]: I would suggest using only one arg for this
```suggestion
    parser.add_argument('-r',
                        '--restrict',
                        action='store',
                        required=False,
                        help="Number of rows to restrict for validation per file. e.g. --restrict 1000 for only validating the first 1000 lines")
```

File: omop_file_validator.py [hunk 2]
```diff
@@ -898,4 +946,22 @@ def evaluate_submission(d):
...
+
+    args = parser.parse_args()
+
+    evaluate_submission(settings.csv_dir,
+                        restrict=args.restrict,
+                        n_rows=args.n_rows)
```
Comment (nishanthpp93) [inline omop_file_validator.py]: You can use only one param here. `args.restrict` will be the number of rows to be validated. If None, skip restriction.
```suggestion
    evaluate_submission(settings.csv_dir,
                        restrict=args.restrict)
```

File: omop_file_validator.py [hunk 3]
```diff
@@ -727,7 +772,7 @@ def run_json_checks(file_path, f):
                                         lambda fmt: date_format_valid(
                                             str(value), fmt), fmts))):
                             e = dict(message=err_msg + ": line number " +
-                                     str(idx + 1),
+                                     str(idx),
```
Comment (nishanthpp93) [inline omop_file_validator.py:751]: Is this going to change the line number?

File: omop_file_validator.py [hunk 4]
```diff
@@ -336,7 +358,11 @@ def check_csv_format(f, column_names):
         line = header
         if header != column_names:
             results.append([header_error_msg, header, column_names])
+
         for idx, line in enumerate(reader, start=2):
+            if restrict and idx - 1 > n_rows:
+                break
```
Comment (nishanthpp93) [inline omop_file_validator.py]: This will become 
```suggestion
            if restrict and idx - 1 > restrict:
                break
```

**Review by nishanthpp93 — CHANGES_REQUESTED** [review]
Suggested some improvements. Let me know if you have any questions.

**Comment by jp3477** [comment]
@nishanthpp93 Sorry it took me so long to get to this. I removed the n_rows argument. Can you explain your question about the line number though?

File: omop_file_validator.py [hunk 5]
```diff
@@ -390,9 +398,12 @@ def check_json_format(f, column_names):
     idx = 1
 
     try:
-        json_list = list(f)
```
Comment (nishanthpp93) [inline omop_file_validator.py:393]: Nice, this may improve memory utilization

**Review by nishanthpp93 — APPROVED** [review]
Looks good! I would suggest asking a site test this too

**Pull request merged**